lib/vmpooler.rb

```diff
@@ -167,6 +171,8 @@ def self.new_metrics(params)
       Vmpooler::Statsd.new(params[:statsd])
     elsif params[:graphite]
       Vmpooler::Graphite.new(params[:graphite])
+    elsif params[:prometheus]
+      Vmpooler::VmpPrometheus.new(params[:prometheus])
```

Comment:
Non-sarcastic question: why shift from using the metric provider name to a prefixed one? Aka, why `Vmpooler::VmpPrometheus` instead of just `Vmpooler::Prometheus`?

Reply:
Yeah - fixing this - was a stage when I was making lot of changes to try and get it working.

Reply:
Done

Reply:
@genebean As discussed - got a namespace conflict when I renamed it back to ```Prometheus``` so going for a different name altogether like ```Promstats``` and will give the .rb file the same name.